tools/bin/ecflow/processes/main/post_processing/merge_fluxes.py

```diff
@@ -0,0 +1,104 @@
+''' merge_fluxes.py
```

Comment:
This tool is called by percentiles.ecf in ecflow. It merges the fluxes from today with those for the rest of the year leading up to today so that runoff can be summed more easily and fluxes can be archived more easily. The annual files can also be used to get the Oct. 1 SWE for the current water year.